Simulator/Test/test_kinematics.py

```diff
@@ -13,9 +14,21 @@
 
 
 def test_calc_q_dot():
+
+	# test against formula from notes
 	q = quat(np.random.rand(4))
 	w = np.random.rand(3)
 	test1 = .5 * L(q) @ np.concatenate((np.array([0]), w))
 	test2 = calc_q_dot(q, w)
+	np.testing.assert_allclose(test1, test2, atol=tol)
+
+	# trivial case
+	q = quat(np.random.rand(4))
+	w = np.zeros(3)
+	np.testing.assert_allclose(calc_q_dot(q,w), np.zeros(4), atol=tol)
 
-	np.testing.assert_allclose(test1, test2, atol=tol)
\ No newline at end of file
+	# against pyquaternion
+	q = quat(np.random.rand(4))
```

Comment:
Is it possible to repeat these random tests more than just once? Not that I'm concerned about it, but bc we have randomness we aren't sure of the actual numbers, but if we ran 50-100 of these then I wouldn't really be worried at all.

Reply:
Sure, I have no qualms with that. I'll throw it in a for loop.